asmcomp/amd64/emit.mlp

```diff
@@ -464,16 +459,15 @@ let add_float_constant cst =
     lbl
 
 let emit_float_constant f lbl =
-  _label (emit_label lbl);
-  D.qword (Const f)
-
-let emit_global_label s =
-  let lbl = Compilenv.make_symbol (Some s) in
-  add_def_symbol lbl;
-  let lbl = emit_symbol lbl in
-  D.global lbl;
-  _label lbl
+  D.define_label lbl ~typ:NONE;
```

Comment:
So, simply generating "foo LABEL DWORD/QWORD" does not work (the assembler complains); at least for labels in code (jump targets), one needs to use "foo:".  I've tried using "foo:" in code section and "foo LABEL ?DWORD" in data section, and a good fraction of the testsuite passes, except for some non-deterministic segfaults (perhaps depending on where the image is loaded).